src/tools/crossgen2/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -661,6 +661,11 @@ private uint getMethodAttribsInternal(MethodDesc method)
                 result |= CorInfoFlag.CORINFO_FLG_PINVOKE;
             }
 
+            if (method.RequireSecObject)
```

Comment:
Can you put this under `#if READYTORUN`? We killed declarative security long time ago and if we need to keep parts of it alive for CoreCLR, it should be restricted to there.